code/mapping/ext_modules/mapping_common/mask.py

```
@@ -0,0 +1,283 @@
+    if start_dist_from_hero is not None:
+        _, after = split_line_at(line, start_dist_from_hero)
+        if after is None:
+            return None
+        line = after
+
+    curve = curve_to_polygon(line, width)
+    return curve
+
+
+def project_plane(
+    size_x: float, size_y: float, start_point: Point2 = Point2.zero()
```

Ruff (0.8.2)

Do not perform function call `Point2.zero` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)